exercise/templatetags/exercise.py

```diff
@@ -128,6 +128,33 @@ def submission_status(status):
     return Submission.STATUS[status]
 
 
+"""
+    Used for adding relevant classes to the
+    points_badge to enable textual status.
+"""
+@register.simple_tag(takes_context=True)
+def get_points_class(context):
+    html_class = "" # holds result
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        html_class = "aplus-points"
+        if (context['max'] == 2 and context['points'] == 2)\
+        or context['max'] == 1 and context['points']== 1:
+            html_class += " accepted"
+        elif (context['max'] == 2 and context['points'] == 1):
+            html_class += " ok"
+        elif context['points'] == 0 and context['submitted']:
+            html_class += " rejected"
+    return html_class
+
+
+#Adds the content of the points_badge.
+@register.simple_tag(takes_context=True)
+def get_points_text(context):
+    if context['difficulty'] == "-" and context['max'] <= 2:
+        return ""
```

Comment:
This is the place to have translations. Should use the same code as above for the class. E.g.

```
class = get_class(context)
tests = {
 "ok": _("OK"),
 "accepted": _("Accepted"),
 "rejected": _("Rejected"),
}
return texts[class]
```